service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/DraftOrdersControllerTest.java

```diff
@@ -216,6 +263,92 @@ void aboutToSubmitShouldPopulateHiddenCCDFieldsInStandardDirectionOrderToPersist
         }
     }
 
+    @BeforeEach
+    void setup() {
+        given(mapperService.mapObject(Mockito.any(), Mockito.any()))
```

Comment:
I don't like this mocking of objectMapper. Could you try adding this to the class
```
@ContextConfiguration(classes = {JacksonAutoConfiguration.class})
```
and then use 

```
@Autowired
private ObjectMapper mapper;
```

We should be able to use the real objectMapper here 🤔

Reply:
I agree with Sam on this one, always nice to use real ObjectMapper it doesn't cost much!